airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/message/DestinationMessageQueues.kt

```diff
@@ -26,18 +26,32 @@ interface Sized {
  */
 sealed class DestinationRecordWrapped : Sized
 
+sealed class DestinationFileWrapped : Sized
```

Comment:
No change required now, but I am realizing that we can simplify this by just putting Records or Files on the queues, moving EOS to the StreamAffinedLevel, and letting the consumers interpret the end-of-flow as end-of-stream. (See "POSSIBLE REVISION" at the bottom [here](https://whimsical.com/destinations-bulk-load-cdk-input-message-hierarchy-EfMmUxLyGnxfdov9WEoUyd))

It will take a little refactoring, I'll add it to the list of possible tweaks.

+ @edgao